__tests__/components/ui/SideNavigation/_partials/Action.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { ActivableLink } from "@/components/ui/SideNavigation/_partials/Action";
+import "@testing-library/jest-dom";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+const items = {
+  subitems: [{ label: "subitem", link: "subitem link" }],
+  link: "/dummy",
+};
+describe("Activable link", () => {
+  it("should render the link with subitems", () => {
+    render(<ActivableLink item={items}>Link test</ActivableLink>);
+    const link = screen.getByTestId("activableLinkId");
+    expect(link).toBeInTheDocument();
+    expect(link.textContent).toBe("Link test");
+  });
+
+  it("should render the link with activable link style", () => {
+    render(
+      <ActivableLink item={items} isActive>
+        Link test
+      </ActivableLink>
+    );
+    const link = screen.getByTestId("activableLinkId");
+    expect(link.children[0].tagName).toBe("SPAN");
+  });
```

Comment:
The test description does not explain the code tested here,  would you check this again?